src/model/docker.ts

```diff
@@ -1,13 +1,15 @@
+import { existsSync, mkdirSync } from 'fs';
 import ImageTag from './image-tag';
 import { exec } from '@actions/exec';
+import path from 'path';
 
 const Docker = {
   async build(buildParameters, silent = false) {
-    const { path, dockerfile, baseImage } = buildParameters;
+    const { path: buildPath, dockerfile, baseImage } = buildParameters;
     const { version } = baseImage;
 
     const tag = new ImageTag({ version });
-    const command = `docker build ${path} \
+    const command = `docker build ${buildPath} \
```

Comment:
buildPath is implied by using it in `Docker.build` method. We can leave it as `path` instead.

Reply:
I would,  but ESlint complains that "path" shadows the imported "path" module.  That is the only reason I changed this,  to make ESlint happy.